src/app/components/profile/modal/skills/skills.scss

```diff
@@ -50,6 +50,11 @@
                 font-weight: bold;
                 flex-basis: 100%;
             }
+            
+            #expertise {
```

Comment:
Dont use id's, build on the class instead (since we are using scss you just nest it to be specific